components/match2/wikis/valorant/legacy/match_maps_legacy.lua

```diff
@@ -76,24 +77,19 @@ function MatchMapsLegacy._handleMaps(args)
 			return false
 		end
 
-		local score1
-		local score2
 		if Logic.isNotEmpty(score) then
 			local splitedScore = mw.text.split(score, '-')
-			score1 = splitedScore[1]
-			score2 = splitedScore[2]
-		end
-		if not score1 and not score2 and winner ~= SKIP then
-			local winnerInt = tonumber(winner)
-			score1 = winnerInt == 1 and DEFAULT_WIN or DEFAULT_LOSS
-			score2 = winnerInt == 2 and DEFAULT_WIN or DEFAULT_LOSS
+			args[prefix .. 'score1'] = mw.text.trim(splitedScore[1] or '')
+			args[prefix .. 'score2'] = mw.text.trim(splitedScore[2] or '')
 		end
 
-		args[prefix .. 'score1'] = mw.text.trim(score1 or '')
-		args[prefix .. 'score2'] = mw.text.trim(score2 or '')
 		args[prefix .. 'finished'] = (winner == SKIP and SKIP) or
 			(not Logic.isEmpty(winner) and 'true') or 'false'
 
+		if Logic.isNumeric(winner) or winner == DRAW then
+			args[prefix .. 'winner'] = winner
```

Comment:
Can we map `winner=draw` to `winner=0`?

Reply:
Match group input can handle that no?

Reply:
we want to only support 0 for draw winner input

Reply:
some bot runs will be needed then

Reply:
already did them on sc(2) fwiw